src/transport_none.h

```diff
@@ -55,6 +55,11 @@ typedef int shm_internal_datatype_t;
 typedef int shm_internal_op_t;
 typedef int shmem_transport_ct_t;
 
+struct shmem_transport_ctx_t{ int dummy; };
+
+typedef struct shmem_transport_ctx_t shmem_transport_ctx_t;
+extern void * SHMEM_CTX_DEFAULT;
```

Comment:
Delete me -- this shadows the declaration in ```shmem.h``` and has the wrong visibility.

Reply:
Looks like this is the cause of the Travis testing failures.

Reply:
Oh yeah, the problem is in transport_none - at least I can reproduce it now.  

But when I remove that line (61) I still get the undefined reference in Ubuntu locally...  I'll try a push through Travis anyway.

Reply:
If I give it that visibility attribute, however, all is good...  Is that what this needs?

Reply:
Maybe ```#include "shmem.h"``` in ```transport_none.c```?  Both of the other transports do this, so they are pulling in the declaration with the visibility attribute.

Reply:
Oh jeez, that's got to be it.  Thanks!